types/three/src/loaders/ObjectLoader.d.ts

```diff
@@ -26,17 +26,18 @@ export class ObjectLoader extends Loader {
     parse<T extends Object3D>(json: any, onLoad?: (object: Object3D) => void): T;
     // tslint:disable-next-line:no-unnecessary-generics
     parseAsync<T extends Object3D>(json: any): Promise<T>;
-    parseGeometries(json: any): { [key: string]: InstancedBufferGeometry | BufferGeometry }; // Array of BufferGeometry or Geometry or Geometry2.
-    parseMaterials(json: any, textures: { [key: string]: Texture }): Material[]; // Array of Classes that inherits from Matrial.
-    parseAnimations(json: any): AnimationClip[];
-    parseImages(json: any, onLoad?: () => void): { [key: string]: Source };
-    parseImagesAsync(json: any): Promise<{ [key: string]: Source }>;
-    parseTextures(json: any, images: any): Texture[];
+    parseGeometries(json: Record<string, any>): { [key: string]: InstancedBufferGeometry | BufferGeometry };
+    parseMaterials(json: Record<string, any>, textures: { [key: string]: Texture }): Record<string, Material>;
+    parseAnimations(json: Record<string, any>): Record<string, AnimationClip>;
+    parseImages(json: Record<string, any>, onLoad?: () => void): { [key: string]: Source };
+    parseImagesAsync(json: Record<string, any>): Promise<{ [key: string]: Source }>;
+    parseTextures(json: Record<string, any>, images: any): Record<string, Texture>;
```

Comment:
Just spot-checking this against source, it seems like json is treated as an array [here](https://github.com/mrdoob/three.js/blob/17cde661d11a76d663c92d5489fd1b83df36ae35/src/loaders/ObjectLoader.js#L575-L605).